chsdi/static/doc/source/releasenotes/index.rst

```diff
@@ -14,6 +14,39 @@ Release Notes
 
     <p><a class="reference external" href="rss2.xml"> <i class="fa fa-rss"> RSS Feeds </i></a></p>
 
+.. _releasenotes_20161005:
+
+Release 20161005 - Wednesday, October 5st 2016
+-------------------------------------------------
+
+API & applications
+******************
+
+`API <//api3.geo.admin.ch>`__
+'''''''''''''''''''''''''''''
+- Bug Fixes
+- `Full changelog <https://github.com/geoadmin/mf-chsdi3/compare/r_160921...r_161005>`__
+
+
+`MAP <//map.geo.admin.ch>`__
+'''''''''''''''''''''''''''''
+- Bug Fixes
+- `Full changelog <https://github.com/geoadmin/mf-geoadmin3/compare/r_160921...r_161005>`__
+
+
+Geodata
+*******
++--------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| New    | `Axis of national routes <//map.geo.admin.ch/?layers=ch.astra.nationalstrassenachsen>`__ (ch.astra.nationalstrassenachsen)                                                                           |
```

Comment:
You need to specify the topic here as well, as this is not in default ech topic.